dspace-server-webapp/src/main/java/org/dspace/app/rest/model/UploadAccessConditionDTO.java

```diff
@@ -0,0 +1,101 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.rest.model;
+
+import java.util.Date;
+import java.util.UUID;
+
+import org.dspace.app.rest.model.step.UploadBitstreamRest;
+
+/**
+ * The UploadAccessConditionDTO is a partial representation of the DSpace
+ * {@link ResourcePolicyRest} as used in the patch payload for the upload
+ * submission section (see {@link UploadBitstreamRest}. The main reason for this
+ * class is to have a DTO to use serialize/deserialize the REST model, that
+ * include reference to the GroupRest and EPersonRest object, in the upload
+ * section data in a simpler way where such reference are just UUID. Indeed, due
+ * to the fact that the RestModel class are serialized according to the HAL
+ * format and the reference are only exposed in the _links section of the
+ * RestResource it was not possible to use the {@link ResourcePolicyRest} class
+ * directly in the upload section
+ *
+ * @author Mykhaylo Boychuk (mykhaylo.boychuk at 4science.it)
+ */
+public class UploadAccessConditionDTO  {
```

Comment:
Based on the new JavaDocs here & the improvements to other Javadocs, I would say this class might be better named `UploadBitstreamAccessConditionDTO`, as it's really specific to *Bitstreams* and it's used/needed directly by `UploadBitstreamRest` (so it'd be nice to name them similarly)